### DISCUSSION TITLE
framerate issues when using dual monitors that have different refresh rates

### DISCUSSION THREAD

**Discussion by AllieTheFox** [Maplespe/DWMBlurGlass#164] [Q&A]
Quick bug report, I've been having an issue with my dual monitor setup with mismatched refresh rates (Primary 75hz, secondary 60hz), where window animations will have severely reduced framerate occasionally when moving between windows on different monitors. Video included shows the behavior.

**Comment by ALTaleX531** [comment]
Although you haven't attached the video yet, I'm sure it's a Windows issue.

**Reply by not-nullptr** [reply]
unsure about this. i run 3 monitors; 144hz, 390hz and 60hz and windows drag and animate at their correct framerates without tearing.

**Comment by not-nullptr** [comment]
update: i get what you mean. from studying the behaviour, it appears dwmblurglass limits the framerate of the blur effect *unless you're dragging the window* in order to save resources
